src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_access_restriction_commands.py

```diff
@@ -124,7 +124,7 @@ def test_webapp_access_restriction_add_ip_address_validation(self, resource_grou
             JMESPathCheck('[1].ipAddress', '2004::1000/128')
         ])
 
-    @ResourceGroupPreparer()
+    @ResourceGroupPreparer(location='centralus')
```

Comment:
why need add location this time ?

Reply:
The default location should be westus I think. In our sub, the number of web app has exceed the max possible value. I cannot create any more appservice.

Reply:
But when I tried to query the number of app service in our sub, I cannot find those app service which is strange.

Reply:
it is because for default location (westus) there is no enough resources to use.